modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/planner/PlannerTest.java

```diff
@@ -1299,4 +1308,110 @@ public void testNotStandardFunctions() throws Exception {
             checkSplitAndSerialization(phys, publicSchema);
         }
     }
+
+    /** */
+    @Test
+    public void testLongPlanningTimeout() throws Exception {
```

Comment:
`PlannerTest` already has a lot of unsorted tests, let's move this test to another class.